README.md

```diff
@@ -62,6 +62,12 @@ Instead, it will copy all the configuration files and the transitive dependencie
 
 You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
 
+## Unit Test
```

Comment:
npm test already has a section above. just modify that section to add this content to it.

Reply:
OK